modular_skyrat/modules/ices_events/code/ICES_event_config.dm

```diff
@@ -548,6 +548,7 @@
 /datum/round_event_control/spacevine
 	max_occurrences = 2
 	weight = MED_EVENT_FREQ
```

Comment:
Reverting the modular changes is nice. However if you want it to run more frequently this needs to be changed to HIGH_EVENT_FREQ.

Reply:
i don't think we'd want it rolling more then grid-check 
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/62520989/288fc798-3026-49b4-9d98-74bac581818f)

Reply:
Nah, I don't think so either, it was just mostly caused by the very steep min_players requirement.

Reply:
I would change the title and PR description as this PR will not be allowing vines to run more frequently.

Reply:
The title doesn't say that they're going to roll more frequently. And they _will_ run more frequently, because now they'll be able to run during lowpop, which they couldn't previously.